client/dashboard/me/billing-purchases/urls.ts

```diff
@@ -0,0 +1,18 @@
+import type { Purchase } from '../../data/purchase';
+
+export function getPurchaseUrl( purchase: Purchase ) {
+	return getPurchaseUrlForId( purchase.ID );
+}
+
+export function getPurchaseUrlForId( id: number | string ) {
+	if ( ! id ) {
+		// eslint-disable-next-line no-console
+		console.error( 'Cannot display manage purchase page for subscription without ID' );
+		throw new Error( 'Cannot display manage purchase page for subscription without ID' );
+	}
+	return `/v2/me/billing/purchases/purchase/${ id }`;
```

Comment:
I believe we shouldn't need to use `/v2` prefix anywhere in HD. As long as we're using the correct router link, the URL will be resolved correctly.